objects/client.c

```diff
@@ -2834,25 +2779,6 @@ luaA_client_get_icon(lua_State *L, client_t *c)
 }
 
 static int
-luaA_client_get_focusable(lua_State *L, client_t *c)
-{
-    bool ret;
-
-    if (c->focusable_set)
-        ret = c->focusable;
-
-    /* A client can be focused if it doesnt have the "nofocus" hint...*/
-    else if (!c->nofocus)
-        ret = true;
-    else
-        /* ...or if it knows the WM_TAKE_FOCUS protocol */
-        ret = client_hasproto(c, WM_TAKE_FOCUS);
```

Comment:
We had size hints in Lua once. It caused problems (but I don't remember which ones) and was eventually moved back.
As for moving the tag-stuff to Lua: I'd love to have that, it would get rid of about half of `ewmh.c`. However, some of the kinds of properties required for this can not even be set via our `xproperty` code (by the way, do we still need that? should it be dropped to simplify our code base?)